i18n/fr/docusaurus-plugin-content-docs/current/miscl/trickery.md

```diff
@@ -0,0 +1,39 @@
+---
+sidebar_position: 2
+---
+
+# 🟢 \*\*\*\*Détection de tricherie
```

Comment:
`\*\*\*\*` ?

Reply:
whoops, sorry, totally a case of tunnel vision! good catch. i'll fix the other instances too